asf_heat_pump_suitability/getters/get_target.py

```diff
@@ -1,8 +1,7 @@
 import polars as pl
-import warnings
+import polars.selectors as cs
 
 from asf_heat_pump_suitability import config
-from asf_heat_pump_suitability.getters import base_getters
 
 
 def get_df_target_nrooms() -> pl.DataFrame:
```

Comment:
Good spot thank you! At the moment we don't use the `get_df_target_nrooms()` function at all but I left it in the code from when I originally wrote it in case we wanted to implement it at a later point. I will add a #  TODO note to flag that Scotland would need to be added if we do use it.

Reply:
Makes sense